app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -142,7 +142,7 @@ class WalkthroughActivityTest {
   }
 
   @Test
-  @DisableAccessibilityChecks
+  @DisableAccessibilityChecks // Disabled as this is a test file and will not be used by user
```

Comment:
This is not a test file. This is test file for `WalkthroughActivity` so please add correct TODO. In this entire file

Reply:
@vrajdesai78 This is pending